src/main.rs

```diff
@@ -11,10 +11,20 @@
 // We won't use the standard library because it requires OS abstractions like threads and files and
 // those are not available in this platform.
 #![no_std]
+#![feature(alloc)]
+
+extern crate alloc;
+extern crate freertos_rs;
+extern crate freertos_alloc;
 
 mod cc3200;
 use cc3200::{Board, Utils, LedEnum, LedName};
 
+use alloc::arc::Arc;
+use freertos_rs::{ CurrentTask, Duration, Task, Queue };
```

Comment:
Nit: ::{CurrentTask, ..., Queue}
I usually use rustfmt to force formatting. Sometimes it's a bit over the top like if you use it on isr_vectors.rs